maps/tradeship/tradeship-0.dmm

```diff
@@ -658,8 +658,14 @@
 /turf/simulated/floor,
 /area/ship/trade/aft_port_underside_maint)
 "bu" = (
-/obj/machinery/door/window/brigdoor/northright,
-/obj/machinery/door/window/brigdoor/southright,
+/obj/machinery/door/window/brigdoor/northright{
+	req_access = list("ACCESS_XENOBIO");
```

Comment:
:disgust: 
Why not set access on the area? Area access inheritance is /door level.

Reply:
Access is set on the area, it just won't inherit it and I couldn't work out why.

Reply:
#1253